### ISSUE TITLE
Please continue this

### ISSUE THREAD

**Issue by shaadart** [Unknown6656/Anti-Social-Chat-Platform#2]
I am really excited for this project, please make it, ❤️ 
If you need help please ask

**Comment by Unknown6656** [comment]
Uh .... are you sure that there is a need/market for it? :P
The project was kind of dogshite, to be honest.

I mean, I can try to work on it, but that will have to wait a couple of weeks as I am currently in the process of moving between two countries.

**Comment by shaadart** [comment]
There is a market man,
People are commiting suicide of because of the vulgarity that current platform supports. 
As you are a programmer, You are designed such a way to make this humaity a safe placce.

BTW, Can I get more information on te project, like how it works, or what is the thing make it work

**Comment by Unknown6656** [comment]
Well, the chat platform was designed to allow user anonymity.
Thus, users are able to post what they want without fear of censoring. Such a platform does not provide safety, but liberty.
I'm not quite sure that these values align well with your goal of creating a safe place 😉

The project is rather straight-forward:
It's a custom HTTP server written in C#, which does not use any existing HTTP library or framework (no ASP.NET etc.). The server supports SSL/TLS, possesses a MySQL database for user/message management <sup>(1)</sup>, a LESS parser, and support for templated HTML documents <sup>(2)</sup>.

I do not think that I have a currently running version of this project. I'd advise you to clone and compile it, and see what happens. I expect that the code needs some major refactorings and an update to .NET6 and C#10 (I believe that it was written for .NETFramework 4.6 and C#7), but maybe you'll get it to run properly.....

(1) see the `dbo.*.sql` files in the folder https://github.com/Unknown6656/Anti-Social-Chat-Platform/tree/master/Server/SQLScripts
(2) template interpolation is performed using the sequence `§ <variable name> §` in a source file, see e.g.: https://github.com/Unknown6656/Anti-Social-Chat-Platform/blob/master/Server/Resources/error.html

**Comment by Unknown6656** [comment]
_(To be honest, I'm not sure how exactly all of the code works, as the project is more than 4 years old .....)_

**Comment by Unknown6656** [comment]
Of course, if you have any code changes or pull requests, feel free to push them and I will merge them into the main branch.
